payments/models.py

```diff
@@ -558,8 +638,14 @@ class OrderLogEntry(UUIDModel, TimeStampedModel):
         related_name="log_entries",
         on_delete=models.CASCADE,
     )
-    status = EnumField(OrderStatus)
+    from_status = EnumField(OrderStatus, blank=True, null=True)
```

Comment:
not sure if it can be blank 🤔

Reply:
I kept it like that in order to be able to migrate the existing `status -> to_status`, so if there's any existing orders, there wouldn't be any "from" status.

Another option would be to set `WAITING` as default `from_status` on the migration and make it non-nullable.

Reply:
Ping @igordavydsson

Reply:
we don't have any orders in prod, so we can maybe set a default on the migration, that would be ok for our test instance 👍